01-messaging.md

```diff
@@ -258,10 +258,19 @@ The `features` field MUST be padded to bytes with 0s.
     1. type: 1 (`networks`)
     2. data:
         * [`...*chain_hash`:`chains`]
-
+    1. type: 3 (`compression_algorithms`)
+    2. data:
+        * [`...*byte`:`supported_algorithms`]
 
 The optional `networks` indicates the chains the node is interested in.
 
+The optional `compression_algorithms` is a bitfield indicating what compression
+algorithms the node supports for [extended gossip queries](07-routing-gossip.md#query-messages).
```

Comment:
Should this be `gossip queries` instead of `extended gossip queries` since this change would impact the SCIDs in normal `reply_channel_range` and `query_short_channel_ids` messages?

Reply:
Good point, I'll fix this.

Reply:
Done in https://github.com/lightningnetwork/lightning-rfc/pull/825/commits/8548f2f90e2459f38f0563c8abd2f46f926f4043